2.0/contrib/etc/puma.cfg

```diff
@@ -1,4 +1,36 @@
+def get_max_memory()
+    if ENV.has_key?('MEMORY_LIMIT_IN_BYTES')
+        return ENV['MEMORY_LIMIT_IN_BYTES'].to_i
+    end
```

Comment:
no, there's no new no limit value, we just added maximum. We don't mind very high values here, because we're maxing out on the number of cores anyway